src/Settings/Builders/Ghc.hs

```diff
@@ -79,6 +79,7 @@ commonGhcArgs = do
             , arg "-hcsuf", arg $ hcsuf way
             , wayGhcArgs
             , packageGhcArgs
+            , arg "-lnuma"
```

Comment:
Shouldn't this be dependent on whether we enable NUMA support in `./configure` ?

Reply:
I supplied no such flag, neither is there any such flag with configure script help. I think it is default setting. Reverting this change and looking further into the matter.